src/PostList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+type Props = {
+  posts: Post[];
+};
+
+const PostsList: React.FC<Props> = ({ posts }) => (
+  <ul>
+    {posts.map((post) => (
+      <li>
+        <b>{post.title}</b>
+        <div>{post.body}</div>
+        <hr />
+      </li>
```

Comment:
If we assume that posts will be one day extended. It would be wise to move this to another component eg. <Post />